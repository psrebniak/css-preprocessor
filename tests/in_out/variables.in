$var: 1;
.test {
    test: ${var};
}
$var: "string with \" symbol ";
.test {
    test: ${var};
}
$selector: container;
.${selector} {
    test: test;
}
$attribute: "attr";
tag[attribute=${attribute}] {
    test: test;
}
