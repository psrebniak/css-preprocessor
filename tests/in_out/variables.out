.test {
    test: 1;
}
.test {
    test: "string with \" symbol ";
}
.container {
    test: test;
}
tag[attribute="attr"] {
    test: test;
}
